src/mmw/requirements/development.txt

```diff
@@ -4,3 +4,4 @@ flake8==4.0.1
 jedi==0.17.2
 ipython==7.17.0
 ipdb==0.13.9
+curlify2==1.0.1
```

Comment:
Don't think we need to change this, but in the future, prefer making dependency changes in their own commits. They often have other implications, and can benefit from isolating.